pyproject.toml

```diff
@@ -27,7 +27,7 @@ classifiers = [
 ]
 dependencies = [
     "anyio>=3.4.0,<5",
-    "typing_extensions>=3.10.0; python_version < '3.10'",
+    "typing_extensions>=4.4.0; python_version < '3.13'",
```

Comment:
`default=` was introduced in this version: https://typing-extensions.readthedocs.io/en/latest/#typing_extensions.TypeVar